components/request/request.html

```diff
@@ -1,36 +1,105 @@
-<div ng-controller="RequestController as model">
-  <div ng-show="model.loginRequired" ng-init="model.redirect()">
-    <h1 align="center">Log in</h1>
-    <form class="form-horizontal" role="form">
-      <fieldset>
-
-        <br-input br-model="model.username"
-          br-options="{name: 'username', label: 'Email'}">
-          Please enter your preferred email address.
-        </br-input>
-
-        <br-input br-model="model.password"
-          br-options="{name: 'password', label: 'Passphrase', type: 'password'}">
-          Please enter your passphrase (for example: 42RhinosStampedeAtTeaTime).
-        </br-input>
-
-        <div style="text-align:center;">
-          <input type="checkbox" name="public-terminal" ng-model="model.publicComputer"/> I'm using a public computer
+<div class="row" ng-controller="RequestController as model">
+  <div ng-show="model.display.identityChooser" ng-init="model.identityChooser()">
+    <h1 class="col-md-offset-3" style="padding-left: 40px">
+      <span ng-show="!model.identitySelected">Select an Identity</span>
+      <span ng-show="model.identitySelected">Log In</span>
+    </h1>
+    <div class="col-md-offset-3 col-md-6">
+      <div ng-hide="model.identitySelected && !identity.selected"
+        class="well" ng-repeat="identity in model.identities">
```

Comment:
Does the identity selector look better if you remove `class="well"` either here or below?